zsh/configs/post/path.zsh

```diff
@@ -1,8 +1,10 @@
 # ensure dotfiles bin directory is loaded first
 PATH="$HOME/.bin:/usr/local/sbin:$PATH"
 
-# load rbenv if available
-if command -v rbenv >/dev/null; then
+# load ASDF, falling back to rbenv if not available
+if [ -d "$HOME/.asdf" ]; then
+  . $HOME/.asdf/asdf.sh
+elif command -v rbenv >/dev/null; then
   eval "$(rbenv init - --no-rehash)"
```

Comment:
Should we add an echo here to encourage folks to move to ASDF or override in their local dotfiles? How long do we keep this fallback?

Reply:
3 months? I'm thinking the same thing about our Vundle -> vim-plug migration. Anyone waiting longer than that to re-sync can take on the burden of resolving conflicts.